Src/AutoFakeItEasy/Properties/AssemblyInfo.cs

```diff
@@ -33,8 +33,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("3.50.2.0")]
-[assembly: AssemblyFileVersion("3.50.2.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyInformationalVersion("1.0.0.0")]
```

Comment:
I believe we don't have too much options here as we will not update the committed `AssemblyInfo` files on each update more. Rather, we temporarily update files immediately before the build and don't expect these changes to be committed. Therefore, there is no way to overcome the issue when you compile your project directly - version will be non-actual in any case.

If we don't reset committed version to `1.0.0.0`, it will lead to even larger confusion. Instead of having the `1.0.0.0` that could indicate that you just have a default version, you will always get `3.50.2` - the latest version which was released using the old release approach.

I'd say that it should not be an issue. We just need to put a notice somewhere in the wiki that you should use FAKE to get assemblies of the desired version.